src/Tests/Collector/MethodCollectorTest.php

```diff
@@ -50,7 +50,7 @@ public function testType()
 
     private function getClassMethod($name)
     {
-        $classMethod = new \StdClass();
+        $classMethod = new \stdClass();
```

Comment:
not wrong per se but will give a warning in any IDE still. As the phpdoc document it as `stdClass` and not `StdClass` I feel like it should be done so as well